kuka_kr4_support/package.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr4_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR 4 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for the KUKA KR 4 manipulators. This currently includes the R540 only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 4 R600 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information
+      in the <em>KUKA Roboter GmbH - KR 3 AGILUS Specification</em> version
+      <em>Datasheet KR 4 R600 V13.1, 19.03.2021</em>.
+      All urdfs are based on the default motion and joint velocity limits,
+      unless noted otherwise (ie: no support for high speed joints,
+      extended / limited motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="simon.schmeisser@optonic.com">Simon Schmeisser (Optonic GmbH)</author>
```

Comment:
```suggestion
  <author>Simon Schmeisser (Optonic GmbH)</author>
  <maintainer email="simon.schmeisser@optonic.com">Simon Schmeisser (Optonic GmbH)</maintainer>
```

You are a maintainer here, so we should list you here.

In those cases where the author is also a maintainer, we don't add the email address to the `author` tag.